explorer/client/src/utils/api/DefaultRpcClient.ts

```diff
@@ -6,11 +6,9 @@ import { JsonRpcProvider } from '@mysten/sui.js';
 import { getEndpoint, Network } from './rpcSetting';
 
 // TODO: Remove these types with SDK types
-export type AddressBytes = number[];
-export type AddressOwner = { AddressOwner: AddressBytes };
+export type AddressOwner = { AddressOwner: string };
```

Comment:
Let's also try to remove this

Reply:
have these types been removed entirely from the system ?